src/lib/arch/aarch64/Architecture.cc

```diff
@@ -1,3 +1,5 @@
+#include "simeng/arch/aarch64/Architecture.hh"
```

Comment:
Why was this only added now? Not to say that it is incorrect, but why wasn't it needed before?

Reply:
Not necessary as included in InstructionMetadata.hh. Probably inserted to help the IDE. Have removed